helm/hpcc/templates/_helpers.tpl

```diff
@@ -855,24 +855,26 @@ Generate list of available services
 - name: {{ $esp.name }}
   class: esp
   type: {{ $esp.application }}
-  port: {{ $esp.servicePort }}
+  port: {{ $esp.service.servicePort }}
   {{- if hasKey $esp "tls" }}
   tls: {{ $esp.tls }}
   {{- else }}
   tls: {{ ($.Values.certificates | default dict).enabled }}
   {{- end }}
-  public: {{ $esp.public }}
+  public: {{ (ne ( include "hpcc.isVisibilityPublic" (dict "root" $ "visibility" $esp.service.visibility))  "") | ternary "true" "false" }}
```

Comment:
trivial: if isVisibilityPublic returned only "true" or "false" (never ""), could you simplify?

Reply:
No - because it would be returning them as strings, not as a boolean